src/firmware/firmwares.json

```diff
@@ -25,5 +25,8 @@
   },
   "l0ud": {
     "SlimeVR-Tracker-ESP-BMI270": ["main"]
+  },
+  "Multi-sensor/Glove": {
```

Comment:
This should be inside the SlimeVR section at the top of the file, the ``{  "OWNER": { "REPO": ["BRANCH", "BRANCH2"] } }``